packages/icons/build-icons.js

```diff
@@ -35,24 +41,30 @@ for (const svgFile of svgFiles) {
     .join('');
 
   const { svg: svgObj } = JSON.parse(xmlParser.toJson(svg));
+
+  const updatedSVG = traverse(svgObj).map(function (x) {
+    if (this.isLeaf && x === 'colorPrimary') this.update(colorPrimary);
+    if (this.isLeaf && x === 'colorSecondary') this.update(colorSecondary);
+    if (this.isLeaf && x === 'colorWhite') this.update(colorWhite);
+    if (this.isLeaf && x === 'colorError') this.update(colorError);
+    if (this.isLeaf && x === 'colorWarning') this.update(colorWarning);
+    if (this.isLeaf && x === 'colorDarkGreyMain') this.update(colorDarkGreyMain);
+  });
```

Comment:
```suggestion

const colorMap = {
colorPrimary: '#0B4566',
colorSecondary: '#FFF',
}

  const updatedSVG = traverse(svgObj).map(function (x) {
  if (!this.isLeaf) return
  if (!Object.keys(colorMap).includes(x)) return
  
  this.update(colorMap[x])

  });
```

Could be less verbose

Reply:
Nice suggestion 👌🏼